docs/manual/manual_english.md

```diff
@@ -0,0 +1,246 @@
+**Controls**
+On this screen, the remote control buttons fulfill the following functions:
+- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+
+
+`` ''
+- Bottom: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- Selection: Short press switches to next screen. If it stays pressed it accesses the configuration screen.
+`` ''
+### Travel time
+
+On this screen you can see the time elapsed since the last reset performed since the settings menu, including the time the motorcycle has been stopped. This is especially useful for regularity rallies.
+**Controls**
+On this screen, the remote control buttons fulfill the following functions:
+- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- **Down**: If QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed it accesses the configuration screen.
+
+### Course
+
+Indicates the current heading in degrees, being 0º North and 180º South.
+**NOTE**: Heading is calculated by drawing vectors between GPS location points, so this value is not valid if the motorcycle is not moving.
+**Controls**
+On this screen, the remote control buttons fulfill the following functions:
+- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- **Down**: If QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed it accesses the configuration screen.
+
+## Setting
+
+This is the screen from which the **Baja Pro** is configured.
+
+### Access the configuration menu
+
+To access the configuration menu, I am **stating at any screen other than the Odometer**, press and hold the **Select** button (rear button of controller) for a few seconds.
+
+### Navigate within settings
+
+To select options within the setup menu using the knob:
+- **Up**: Upper front button
+- **Lower**: Lower front button
+- **Select**: Back button
+- **Cancel**: Press and hold the back button
+- **Exit menu**: Press and hold the back button
+
+### Reset trip
+
+Use this option before starting each stage of a rally. This resets:
+- Total distance
+- Partial distance
+- Average speed
+- Maximum speed
+- Travel time
+
+
+### Language
+
+The **Baja Pro** can be configured in the following languages:
+- Spanish
+- English
+Select the one that best suits your needs.
+
+### Accuracy
+
+The accuracy level of the **Baja Pro** can be set between:
+- **One decimal**: Distances show increments every 100 meters. Upon landing, will add or subtract 100 meters to the partial distance.
```

Comment:
```suggestion
- **One decimal**: Distances show increments every 100 meters. When manually adjusting, it will add or subtract 100 meters to the partial distance.
```